shell/platform/darwin/common/cf_utils.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_DARWIN_COMMON_CF_UTILS_H_
+#define FLUTTER_SHELL_PLATFORM_DARWIN_COMMON_CF_UTILS_H_
+
+#include <CoreFoundation/CoreFoundation.h>
+#include "lib/ftl/macros.h"
+
+namespace shell {
+
+template <class T>
+class CFRef {
```

Comment:
That's a good idea. I'll extract this out. I am not adding any base references to any new files. hoping to get rid of base entirely at some point. 

fml:: is a good suggestion. But how about this blockbuster series:
base::
base::two::
base::tokyo::drift::
base::four::
.
.
.
fate::of::the::base::
?